### ISSUE TITLE
🐛 [BUG]: Controls do not work in older chromium browsers

### ISSUE THREAD

**Issue by atyshka** [bcakmakoglu/vue-flow#1089]
### Is there an existing issue for this?

- [X] I have searched the existing issues and this is a new bug.

### Current Behavior

Controls do not appear to work in Chromium versions 86 and below. I'm not sure if you care about backwards compatibility, but this is a fairly recent version, within the last 3 years. The SVGs render with height 0. I'm not sure why this is happening, since flexbox was fairly mature by then. 

<img width="48" alt="Screen Shot 2023-08-24 at 1 32 33 PM" src="https://github.com/bcakmakoglu/vue-flow/assets/19317207/5bb78a4d-0b9b-40cf-9349-cc09cb0f233a">

### Expected Behavior

The SVGs should render. I was able to get this to work by changing the css for the SVGs from this:
```
.vue-flow__controls-button svg {
  max-width: 12px;
  max-height: 12px;
  overflow: visible;
}
```
to this:
```
.vue-flow__controls-button svg {
  width: 12px;
  height: 12px;
  overflow: visible;
}
```

### Steps To Reproduce

Download Chromium 86 from here: https://chromium.cypress.io 
Visit https://vueflow.dev/examples/horizontal.html. Notice that the SVGs are missing on the controls

**Comment by bcakmakoglu** [comment]
Thanks for the report, I'll check it out when I have some time 😅

**Comment by bcakmakoglu** [comment]
Sorry for the long delay on this one
It'll be fixed in `1.30.0`
